src/threads/MutexManager.cpp

```diff
@@ -9,6 +9,10 @@ namespace threads {
 // created on the fly.
 // ----------------------------------------
 
+void MutexManager::clear() {
+    mutexes.clear();
+}
```

Comment:
A lack of clearing was causing issues when repeatedly running multi-threaded functions with the same module.